flink-runtime/src/test/java/org/apache/flink/runtime/state/ttl/TtlMapStateAllEntriesTestContext.java

```diff
@@ -66,4 +67,9 @@ public void update(Map<Integer, String> map) throws Exception {
 	public Object getOriginal() throws Exception {
 		return ttlState.original.entries() == null ? Collections.emptySet() : ttlState.original.entries();
 	}
+
+	@Override
+	public boolean isOriginalEmptyValue() throws Exception {
+		return !Sets.newHashSet(getOriginal()).equals(emptyValue);
```

Comment:
Just wonder if this method check whether original state is empty value, why we still need to return the opposite value? In other words, shall we just return `Objects.equals(emptyValue, Sets.newHashSet(getOriginal()))`?